zenaton/services/graphql_service.py

```diff
@@ -1,12 +1,25 @@
-from ..exceptions import ExternalError, InternalError
+from ..exceptions import InternalError
 import json
 import requests
 
 
 class GraphQLService:
+
+    # Queries
+    FIND_WORKFLOW = """
+        query ($custom_id: String!, $environment_name: String!, $programming_language: ProgrammingLanguage!, $name: String!) {
+            findWorkflow(custom_id: $custom_id, environment_name: $environment_name, programming_language: $programming_language, name: $name) {
+                id
```

Comment:
We don't return the `id` in php and node. Not sure it is important though ...

Reply:
It's not. We won't return anything to the user in any case.
It's just that GraphQL requires you to select one field. We just picked a different field ^^